src/App.jsx

```diff
@@ -14,70 +16,95 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const checkSelectingGood = (row) => {
+    if (selectedGood) {
+      const selected = document.querySelector(
+        '.has-background-success-light',
+      );
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      if (selected !== row) {
+        selected.classList.remove(
+          'has-background-success-light',
+        );
+      }
+    }
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  const selectItem = (e) => {
+    const row = e.target.closest('tr');
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+    if (!row) {
+      return;
+    }
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+    checkSelectingGood(row);
+    const goodInnerText = row.querySelector('[data-cy="GoodTitle"]').innerText;
+
+    row.classList.toggle('has-background-success-light');
```

Comment:
Avoid using classList.toggle method directly on DOM elements. React state should be the source of truth.